code-scanning/frogbot-fix.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# Read more about Frogbot here - https://github.com/jfrog/frogbot#frogbot
```

Comment:
Kindly add more details around what the template does or what the action does, how it does this and what pre-requisites are there to use it

Reply:
Done :)